src/nvim/edit.c

```diff
@@ -4116,7 +4138,7 @@ static int ins_compl_get_exp(pos_T *ini)
   static buf_T *ins_buf = NULL;          // buffer being scanned
 
   pos_T *pos;
-  char_u **matches;
+  char_u * *matches;
```

Comment:
```suggestion
  char_u **matches;
```

Reply:
That's weird. All these four changes seem to the best of my knowledge applied to the current PR.